internal/streams/api.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/AlexxIT/go2rtc/internal/api"
 	"github.com/AlexxIT/go2rtc/internal/app"
+	"github.com/AlexxIT/go2rtc/pkg/hap"
```

Comment:
I don't like this dependency in this module. The streams module does not need to know about the existence of any particular source.